internal/developer/schema.sql

```diff
@@ -44,3 +44,17 @@ ALTER TABLE developer_profile ADD COLUMN search_status VARCHAR(20) NOT NULL DEFA
 ALTER TABLE developer_profile ADD COLUMN role_types VARCHAR(20) NOT NULL DEFAULT 'full-time';
 ALTER TABLE developer_profile ADD COLUMN detected_location_id VARCHAR(255) DEFAULT NULL;
 ALTER TABLE developer_profile_message ADD COLUMN sender_id CHAR(27) DEFAULT NULL;
+
+
+CREATE TYPE valid_developer_metadata_type AS ENUM ('experience', 'education', 'github');
+
+CREATE TABLE IF NOT EXISTS developer_metadata (
+    id SERIAL PRIMARY KEY,
```

Comment:
we try to avoid integer serial IDs. I originally used this when starting out on the project for jobs, but right now we use pretty much char(27) everywhere (ksuid tokens).

Reply:
Got it, just curious, what is the technical reason for not using integer serial ID ? Is it for security reason ? so that people can't guess ID and end up at some URL which they shouldn't ?

Reply:
yes security and performance - sequential auto ids impact db insert performance